supervisor/const.py

```diff
@@ -37,6 +37,7 @@
 DOCKER_NETWORK = "hassio"
 DOCKER_NETWORK_MASK = ip_network("172.30.32.0/23")
 DOCKER_NETWORK_RANGE = ip_network("172.30.33.0/24")
+DOCKER_IMAGE_DENYLIST = ["containrrr/watchtower", "v2tec/watchtower"]
```

Comment:
So... isn't any container, unknown to the Supervisor, making the system unsupported? (according to the ADR)

Reply:
I think we should be fair and only mark things as unsupported if they make issues.

Reply:
If we consider that, we should be splitting this into 2 sections.

1) We've detected unknown, external, containers, this is not officially supported
2) We've detected known, external, containers that are known to cause issues

Both deserve a log message of some sort I think.

Reply:
I agree, 1 should set it as unsupported, 2 should set it as unhealthy

Reply:
2) let's show an error but not set unhealthy